_std/pathfinding.dm

```diff
@@ -215,6 +215,11 @@
 				L.Add(T)
 	return L
 
+var/static/obj/item/card/id/AA = new /obj/item/card/id/captains_spare()
```

Comment:
pretty gross

Reply:
Yeah, I'm not very fond of it either, the underlying proc wants an ID rather than an access level, so that it can best do the calculation for what airlocks it can pass through.
I am welcoming other solutions, do you think making a variant of LinkBlockedWithAccess that does not change access instead would be better?

Reply:
A variant which takes a list of access levels instead of an ID would be neat.